src/middleware/errorHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response, NextFunction } from 'express'
+import { HttpError } from 'http-errors'
+import safeStringify from 'fast-safe-stringify'
+import { HttpExceptionOptions } from '../types/http-exception'
+
+export default async (err: Error, req: Request, res: Response, next: NextFunction): Promise<Response | void> => {
+  if (res.headersSent) {
+    return req.socket.destroy()
+  }
+
+  if (!(err instanceof HttpError)) {
+    return res
+      .status(500)
+      .header('Content-Type', 'application/json; charset=utf-8')
+      .send({ message: 'An unexpected error has occurred. Please try again.' })
```

Comment:
This should use `safeStringify`:
```suggestion
      .send(safeStringify({ message: 'An unexpected error has occurred. Please try again.' }))
```

Reply:
Complete